CODEOWNERS

```diff
@@ -1,3 +1,11 @@
-*       @bardielle
-*       @machacekondra
-*       @mikemorency
+# Default code owners for all files and directories
+*       @bardielle @machacekondra @mikemorency
```

Comment:
@shellymiron not sure you can see this warning or not - there are errors on that file: 
```
Unknown owner on line 5: make sure @shellymiron exists and has write access to the repository
/tests/integration/ @shellymiron @elsapassaro
Unknown owner on line 5: make sure @elsapassaro exists and has write access to the repository
…sts/integration/ @shellymiron @elsapassaro
Unknown owner on line 8: make sure @shellymiron exists and has write access to the repository
…workflows/eco-vcenter-ci.yaml @shellymiron @elsapassaro
Unknown owner on line 8: make sure @elsapassaro exists and has write access to the repository
…-vcenter-ci.yaml @shellymiron @elsapassaro
Unknown owner on line 11: make sure @shellymiron exists and has write access to the repository
/scripts/ @shellymiron @elsapassaro
Unknown owner on line 11: make sure @elsapassaro exists and has write access to the repository
/scripts/ @shellymiron @elsapassaro
```

Reply:
ah I see now
do you have an idea what should I change? those are our correct github usernames

Reply:
@bardielle I think you will need to give us general access to the repo, it seems that until we will not have 
writing access it will always fail. after writing permission is granted the file can restrict the modifications

Reply:
I added you a "write" role but I'm not sure it's enough. 
let me know

Reply:
great! seems to work, you can add @elsapassaro aswell and the errors supposed to be gone